vnext/template/cs-app/proj/MyApp.csproj

```diff
@@ -28,9 +28,6 @@
     <PackageCertificatePassword>password</PackageCertificatePassword>
     <AppxGeneratePrisForPortableLibrariesEnabled>false</AppxGeneratePrisForPortableLibrariesEnabled>
     <LangVersion>7.3</LangVersion>
-    {{#useExperimentalNuget}}
-    <UseExperimentalNuget>true</UseExperimentalNuget>    <!-- This will eventually default to true and this property should be removed -->
```

Comment:
I think I'm missing something:
how will projects that are created with --useExperimentalNuget get this property set to true (other than them going in and adding the value manually)

Reply:
for new projects, the flag is just moved to experimental props file, so they can still change it there. for existing projects, they will need to move the prop there if they want their modules to build against the nuget too.

Reply:
we don't want each project to make it's "own" decision about whether or not to use the nuget - we want it to align with the app, and the only way is a common prop file